public/index.php

```diff
@@ -2,20 +2,28 @@
 require_once '..\bootstrap\bootstrap.php';
 
 
-$klein->respond('/', function() use ($twig, $crud){
+$klein->respond('GET','/', function() use ($twig, $crud){
 	$getAll = $crud->getAll();
 	echo $twig->render('home.twig', array(
 		'getAll' => $getAll
 	));
 });
 
-$klein->respond('/[i:id]', function($request) use($twig, $crud){
+$klein->respond('GET','/[i:id]', function($request) use($twig, $crud){
 	$getOne = $crud->getOne($request->id);
 	echo $twig->render('getOne.twig', array(
 		'getOne' => $getOne
 	));
 });
 
+$klein->respond('POST','/create', function() use ($crud){
```

Comment:
to remain in RESTful mode, rename your route to '/', and by defining the verb POST for this route, your app will know how to handle it